src/main/java/com/zhongan/devpilot/provider/ut/UtFrameworkProviderFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.zhongan.devpilot.provider.ut;
+
+import com.zhongan.devpilot.provider.ut.java.JavaUtFrameworkProvider;
+
+public class UtFrameworkProviderFactory {
+
+    public static UtFrameworkProvider create(String language) {
+        switch (language) {
+            case "java":
+                return new JavaUtFrameworkProvider();
```

Comment:
This will create new object every times, it will waste memory. You may cache the result.